hash.c

```diff
@@ -2150,15 +2152,16 @@ rb_hash_stlike_lookup(VALUE hash, st_data_t key, st_data_t *pval)
 
 /*
  *  call-seq:
- *     hsh[key]    ->  value
+ *    hsh[key] -> value
```

Comment:
I don't know how common it is, but do you think we should avoid abbreviations?

Reply:
That's what's been here in Hash, and 'ary' is seen everywhere over at Array.  I'm going to revise all of Hash, so we could change 'em all as we go.

Btw, Matz has approved me as a committer, so as soon as I figure out the security stuff (SSH, etc.), I'll be merging my own stuff.

Reply:
You should still get reviews from other members of the group.

Reply:
Count on it.  I'd never do otherwise.